src/modules/assessments/components/AssessmentForm.tsx

```diff
@@ -118,16 +122,17 @@ const AssessmentForm: React.FC<Props> = ({
 
   // Choose the FormDefiniton to use for rendering, and filter it down based on client attributes (Data Collected About rules).
   const definition = useMemo(() => {
-    let fd = definitionProp;
-    if (!locked && assessment && !assessment.inProgress) {
-      // When editing a non-WIP assessment, we want to use the most recent version of the Form Definition.
-      // If the most recent version differs from the version that was most recently used to updated it, then 'upgradedDefinitionForEditing' is present.
-      fd = assessment.upgradedDefinitionForEditing || fd;
-    }
+    const fd = locked ? viewingDefinition : editingDefinition;
```

Comment:
This change requires https://github.com/greenriver/hmis-warehouse/pull/4649#discussion_r1733379534